docs/spec/data-plane.md

```diff
@@ -93,17 +93,17 @@ CloudEvents received by Sink MAY have
 
 ### Event reply contract
 
-An event sender supporting event replies SHOULD include a `Prefer: reply` header
-in delivery requests to indicate to the sink that event reply is supported. An
+An event sender supporting event replies SHOULD include a `Prefer: kn-eventing-reply`
+header in delivery requests to indicate to the sink that event reply is supported. An
 event sender MAY ignore an event reply in the delivery response if the
-`Prefer: reply` header was not included in the delivery request.
+`Prefer: kn-eventing-reply` header was not included in the delivery request.
```

Comment:
is it kn we are talking about or CloudEvents? `kn-eventing-reply` -> `cloudevents-reply` ?

Reply:
I like the name you proposed better. Also looping @duglin to ensure there won't be any conflict in the future.

Reply:
I thought cloudevents has no concept and is very unlikely to have a notion of a reply, so why do we think that having a cloudevents prefix here makes sense? Not saying no, just trying to understand the reasoning for it.

Reply:
Guessing at the original reasoning of Scott's comment, but I think it's suggesting to indicate the response format rather than the defining organization.

- `cloudevents-reply` indicates that reply is supported AND should be formatted as a CloudEvent message.
- `kn-eventing-reply` indicates that reply is supported AND the definition of the meaning of this prefer value is owned by Knative Eventing.

Do we want to specify the defining organization or the format in the Prefer value name? The examples of Prefer values from the RFC (https://tools.ietf.org/html/rfc7240#section-5.1) don't have organizational prefixes, but that's ok since they apply generally to all HTTP requests.

I have a slight preference for `kn-eventing-reply` because it indicates to users who see the header on their requests where they should go to understand what it means. `cloudevents-reply` is closer to self-documenting but not quite enough. I don't want the CloudEvents WG to be responsible for redirecting people here when they see this value.

I'm also aware that we are changing an unversioned protocol and protocol versioning is on the roadmap. IMO any long-term considerations about other uses for Prefer headers should be put on hold until versioning arrives.